components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/JSON.kt

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.ktx.kotlin
+
+import org.json.JSONArray
+import org.json.JSONObject
+import java.util.TreeMap
```

Comment:
Let's replace it in Fretboard in the same commit.

We will have to add ktx to the dependencies:

```Groovy
implementation project(':support-ktx')